splash/lua_modules/splash.lua

```diff
@@ -98,7 +98,7 @@ local Splash = {}
 Splash.__index = Splash
 
 function Splash.private_create(py_splash)
-  local self = {args=py_splash.args}
+  local self = {args=py_splash.args, members={}, set_js_enabled=drops_self_argument(py_splash['set_js_enabled'])}
```

Comment:
Could you please split this into multiple lines:

```
local self = {
    args=py_splash.args,
    members={},
    ... 
}
```

The line is too long :) there is no pep8 for Lua code, but it is better to be consistent when possible.